common/static/common/js/discussion/views/thread_response_view.js

```diff
@@ -181,6 +182,18 @@
                 return view;
             };
 
+            ThreadResponseView.prototype.toggleCommentSubmitButton = function(event) {
+                var id = event.target.id.split('-').slice(-1)[0]
```

Comment:
Please add semicolons at the end of lines.

Reply:
done